### ISSUE TITLE
Missing (/=) for some ESMF Types

### ISSUE THREAD

**Issue by tclune** [esmf-org/esmf#141]
The following types have (==) defined, but not (/=):

- ESMF_Region_Flag
- ESMF_TermOrder_Flag

Easy to workaround of course, so low priority.

Now I just have to figure out how to tell if two ESMF_DynamicMask objects are the same ...

**Comment by theurich** [comment]
Yes, that is correct. I just compiled the list for #138, and noticed the same. The missing (/=) are from the overloads in file `src/Infrastructure/Util/src/ESMF_UtilTypes.F90`. The information under #138 should help identify what is missing in case you need it.
